client/src/components/ForceGraph.vue

```diff
@@ -653,6 +519,29 @@ export default {
   animation: selected 0.5s 4 alternate ease-in-out;
 }
 
+#grid .veryImportant {
+  stroke-width: 1px;
+  animation: evenMoreImportant 2s infinite ease-in-out;
+}
+
+@keyframes evenMoreImportant {
```

Comment:
as long as it's funny, connected with the commit name that introduces that i would suggested making this name (and the `.veryImportant` on the top more descriptive

Reply:
Different color palette option has been removed. Looks nice, but is not very readable